modules/shared_data/src/main/scala/org/elpaca_metagraph/shared_data/combiners/Combiner.scala

```diff
@@ -117,7 +118,16 @@ object Combiner {
         ).map(_.asInstanceOf[DataSource])
 
         (DataSourceType.Streak, updatedStreakDataSource)
+
+      case update: YouTubeUpdate =>
```

Comment:
It's not really necessary, just a nitpick to keep the pattern, extract the updateYoutubeState to a variable like the others =]